bindings/python/Makefile.am

```diff
@@ -0,0 +1,32 @@
+SUBDIRS = . tests
+
+AM_CXXFLAGS = -Wall
+
+BUILT_SOURCES = $(srcdir)/pire_pywrap.cpp
+SWIG_SOURCES = \
+	$(srcdir)/common.i \
+	$(srcdir)/fsm.i \
+	$(srcdir)/lexer.i \
+	$(srcdir)/scanners.i \
```

Comment:
We need to make `pire_pywrap.cpp` dependent on `*.i`.
Yes, headers is probably more appropriate category